step-capstone/src/components/AddEvent.js

```diff
@@ -1,60 +1,72 @@
 import React, { useEffect } from 'react'
 import { useState } from 'react'
 import {
-    TextField,
-    Grid,
-    Checkbox,
-    FormControlLabel,
+ TextField,
+ Grid,
+ Checkbox,
+ FormControlLabel,
 } from '@material-ui/core';
 import {
-    MuiPickersUtilsProvider,
-    DateTimePicker
+ MuiPickersUtilsProvider,
+ DateTimePicker
 } from '@material-ui/pickers';
 import DateFnsUtils from "@date-io/date-fns";
-
-export default function AddEventHotel(props) {
+export default function AddEvent(props) {
     let overwriting = props.data !== undefined;
-
     // Sets values to previous values if editing, otherwise blank slate
-    const [startDate, handleStartChange] = useState(overwriting ? props.data.startDate : new Date());
-    const [endDate, handleEndChange] = useState(overwriting ? props.data.endDate : new Date());
+    const [startDate, setStartDate] = useState(overwriting ? props.data.startDate : props.startDate);
+    const [endDate, setEndDate] = useState(overwriting ? props.data.endDate : props.startDate);
```

Comment:
This looks odd since you use start date here as well, should it be `const [endDate, setEndDate] = useState(overwriting ? props.data.endDate : props.endDate);`? This applies to the file below as well.

Reply:
To clarify, the variable `props.startDate` is the overall trip's startDate. When user adds a new item (the `props.data` will be undefined), the AddEvent (same for AddFlight and AddHotel) will initialized the Date to be the startDate of the trips instead. I think it is more natural for a event to end in one day, so I initialized both startDate and endDate equal to the overall trip's startDate.